.circleci/config.yml

```diff
@@ -143,16 +143,16 @@ jobs:
           name: '`yarn workspace framer build` changes committed?'
           command: git diff --exit-code
       - run:
-          name: Lint
-          command: yarn lint:ci
+          name: Eslint
```

Comment:
same here. This might be misleading in the future. The package name doesn't matter. Just that we Lint.

Reply:
No preference, we started running the different lint command in isolation in the CI (es, json, style), I thought it would make sense to be all-in in the approach. But we could also bundle all the lint steps into as single command.

Reply:
The preference is to not change things just to change them.

Reply:
So we should merge the Jsonlint step inside the Lint step?

Reply:
we can. it don't see any necessity to change anything though.

Reply:
I have reverted